types/chart.js/index.d.ts

```diff
@@ -29,7 +29,7 @@
 //                 Mathias Helminger <https://github.com/Ilmarinen100>
 //                 Mostafa Sameti <https://github.com/IVIosi>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
```

Comment:
The scriptable part looks good, but I'm not so sure about changing the typescript version.  
It affects multiple packages, and maybe not everybody can use it.  
Why is it needed?